package.json

```diff
@@ -70,9 +70,11 @@
     ]
   },
   "ava": {
+    "babel": true,
     "verbose": true
```

Comment:
We don’t need Babel right? Can this be disabled?

Reply:
Done.